src/packages/ThemeGenerator/ThemeGenerator.ts

```diff
@@ -2,28 +2,92 @@
  * @module ThemeGenerator is responsible for generating
  * custom CSS properties from theme variables.
  */
+import FileSaver from 'file-saver';
+import { ThemeType } from '../../themes/OneUITheme';
 
-type CSSVars = {
+type CssVars = {
     [key: string]: string;
 };
 
-type Theme = (cssVars: CSSVars) => CSSVars;
+type ThemeInfo = {
+    /** unique theme name typed by user */
+    id: string;
+    /** timestamp when theme was created */
+    created: string;
+    /** version of Theme JSON file */
+    themerollerVersion: string;
```

Comment:
According to Ruud,` version` is a misleading name, because users can think that `version` is the version on theme changes. So, I decided to rename it to `themerollerVersion`.